EnhancementofUnderwaterImages/getGBTransmission.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+import math
+
+
+def getGBTransmissionESt(transmissionR,AtomsphericLightTM):
+    depth_map = np.zeros(transmissionR.shape)
+    for i in range(0,transmissionR.shape[0]):
+        for j in range(0, transmissionR.shape[1]):
+            depth_map[i,j]  = math.log(transmissionR[i,j],0.82)
+            # if(depth_map[i,j]>15):
+            #     depth_map[i, j] = 15
+            # if (depth_map[i, j] < 1):
+            #     depth_map[i, j] = 1
+
+
+
+    transmissionG = 0.93 ** depth_map
+    transmissionB = 0.95 ** depth_map
+    # transmissionG = np.zeros(transmissionR.shape)
```

Comment:
explain why you'd comment out these lines and in which situation the user may uncomment and use them